src/background/index.js

```diff
@@ -188,7 +191,7 @@ chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
     }
     tabStats.set(tabId, stats);
 
-    if (!options.trackerWheelDisabled) {
+    if (store.ready(options) && !options.trackerWheelDisabled) {
```

Comment:
what if store is not ready?

Reply:
I am preparing the fix of the hybrids store bug, so that case won't be necessary here anymore.